<launch>
  <param name="point_cloud/rotate_image" value="true"/>
  <param name="object_rec/plane_detection/depth_threshold" value="1.3"/>
  <param name="object_rec/plane_detection/distance_threshold" value="0.03"/>
  <param name="object_rec/plane_detection/ransac_n" value="3"/>
  <param name="object_rec/plane_detection/num_iterations" value="1000"/>
  <param name="object_rec/pointcloud_clustering/eps" value="0.01"/>
  <param name="object_rec/pointcloud_clustering/min_points" value="10"/>
  <param name="object_rec/pointcloud_clustering/rect_min" value="20"/>
  <param name="object_rec/pointcloud_clustering/rect_max" value="100"/>
  <param name="object_rec/show_result" value="true"/>
  <node pkg="ros_vision" type="tabletop_object_recognition.py" name="tabletop_object_recog" cwd="node" output="screen" />
</launch>
